packages/files-ui/src/Components/Modules/FileBrowsers/MoveFileModal.tsx

```diff
@@ -95,25 +94,40 @@ const MoveFileModule = ({ filesToMove, modalOpen, onClose, onCancel, mode }: IMo
   )
 
   const getFolderTreeData = useCallback(async () => {
-    const bucket = buckets.find((bucket) => bucket.type === "csf")
     if (!bucket) return
     filesApiClient.getBucketDirectoriesTree(bucket.id).then((newFolderTree) => {
       if (newFolderTree.entries) {
-        const folderTreeNodes = [
-          {
-            id: "/",
-            title: "Home",
-            isExpanded: true,
-            expandable: true,
-            tree: mapFolderTree(newFolderTree.entries)
-          }
-        ]
-        setFolderTree(folderTreeNodes)
+        if (bucket.type === "share") {
+          // share folder tree
+          const shareFolderTreeRoot = newFolderTree.entries[0].entries[0]
+          const folderTreeNodes = [
+            {
+              id: "/",
+              title: bucket.name || "Home",
```

Comment:
I think we should have something like the bucket id there, or "Root - {bucket id}". But this should be reflected also on the title etc, so prob not for this PR.

```suggestion
              title: bucket.name || "Root",
```